sfa_dash/templates/forms/site_form.html

```diff
@@ -110,7 +112,8 @@
       </div>
       <div class="form-element">
         {{ form.input('Ground coverage ratio', 'number', 'ground_coverage_ratio',
-                      help_text='Azimuth angle of the tracker axis, degrees East of North.', form_data=form_data,
+                      help_text='Ratio of total width of modules on a tracker to the distance between tracker axes. For example, for trackers each with two modules of 1m width each, and a spacing between tracker axes of 7m, the ground coverage ratio is 0.286(=2/7).',
```

Comment:
```suggestion
                      help_text='Ratio of total width of modules on a tracker to the distance between tracker axes. For example, for trackers with two modules of 1m width, and a spacing between tracker axes of 7m, the ground coverage ratio is 0.286(=2/7).',
```

Reply:
Debatable but I think it makes more sense without each each

Reply:
Agreed.